common/scala/src/main/scala/whisk/core/containerpool/logging/SplunkLogStore.scala

```diff
@@ -60,14 +60,24 @@ case class SplunkLogStoreConfig(host: String,
                                 username: String,
                                 password: String,
                                 index: String,
+                                logTimestampField: String,
```

Comment:
It may be helpful to have these config documented in `application.conf` as commented out config